lms/templates/shoppingcart/receipt.html

```diff
@@ -293,7 +292,7 @@ <h3 class="text-center">${_("Please send each professional one of these unique r
         <div class="clearfix">
           <div class="image">
             <img class="item-image" src="${course_image_url(course)}"
-                 alt="${course.display_number_with_default | h} ${get_course_about_section(request, course, 'title')} Image"/>
+                 alt="${course.display_number_with_default | h} ${course.display_name_with_default | h} Image"/>
```

Comment:
Because display_name_with_default is currently already escaped, this is actually double escaping, and the " | h" should probably be removed.  Note: not both of them.